src/index.js

```diff
@@ -29,36 +29,54 @@ import './images/step-icon.png'
 import './images/trend-icon.png'
 import './images/water-icon.png'
 
+let currentUserID;
 
-  let userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData').then(response => response.json()).then(json => json.userData)
-  // console.log(userData)
-  let sleepData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData').then(response => response.json()).then(json => json.sleepData)
-  let hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData').then(response => response.json()).then(json => json.hydrationData)
-  let activityData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData').then(response => response.json()).then(json => json.activityData)
+$('nav').hide()
+$('header').hide()
+$('main').hide()
 
-  let randomId, stats, userRepository, hydrationRepository, sleepRepository, activityRepository, user;
+$('.splash__input--user').keyup((e) => {
+  e.preventDefault();
+  if ($('.splash__input--user').val() !== '') {
+    $('.splash__button').prop('disabled', false);
+  }
+})
 
- Promise.all([userData, sleepData, hydrationData, activityData])
-  .then(data => {
-    userData = data[0];
-    sleepData = data[1];
-    hydrationData = data[2];
-    activityData = data[3];
-  })
+$('.splash__button').on('click', (e) => {
+  var currentUserID = $('.splash__input--user').val();
+  $('.splash__container').hide();
+  $('nav').show();
+  $('header').show();
+  $('main').show();
+  e.preventDefault();
+});
 
-  .then(data => {
-    randomId = Math.floor(Math.random() * (50 - 1) + 1);
-    stats = new Stats(userData, randomId);
-    userRepository = new UserRepository(userData, randomId);
-    hydrationRepository = new HydrationRepository(hydrationData, randomId);
-    sleepRepository = new SleepRepository(sleepData, randomId);
-    activityRepository = new ActivityRepository(activityData, randomId);
-    user = new User(userRepository.getUserData());
-    startApp()
-  })
-  .catch(error => console.log(error))
+let stats, userRepository, hydrationRepository, sleepRepository, activityRepository, user;
+
+let userData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData').then(response => response.json()).then(json => json.userData)
+// console.log(userData)
+let sleepData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData').then(response => response.json()).then(json => json.sleepData)
+let hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData').then(response => response.json()).then(json => json.hydrationData)
+let activityData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData').then(response => response.json()).then(json => json.activityData)
 
+Promise.all([userData, sleepData, hydrationData, activityData])
```

Comment:
@trishalanglois I was able to figure out that our `promiseAll` is being invoked on page load and the value of `currentUserId` isn't set, which is why our values are all undefined.

Reply:
Ahhh, I see.  So, even though we're reassigning the variable on line 45 with the event listener, we need something to reload the page/DOM, correct?
@allisonjw, it seems like this kind of defeats the purpose of why we were using the splash page.  Thoughts?  Should we scrap it and just have a login thing on the main page?  Or we can change the page load to happen when the user clicks on the splash page button.  I think my preference would be the latter since we already have the HTML and some of the CSS done for the splash page.